libbpfgo.go

```diff
@@ -531,6 +531,47 @@ func CreateMap(mapType MapType, mapName string, keySize, valueSize, maxEntries i
 	}, nil
 }
 
+/*
+GetMapByName get an existing map from userspace.
```

Comment:
Could you use // in the function godoc like the other ones ?

nit: "GetMapByName gets an existing map..." (verb)
nit: "can be used for populating BPF..." (verb)
nit: "... or hasH of maps" (typo)